src/router.js

```diff
@@ -3,6 +3,7 @@ var links = require('./links')
 var flatten = require('./flatten')
 var createHistory = require('./history')
 var defaultQs = require('./qs')
+var noop = require('./noop')
```

Comment:
could you inline the function, like `var noop = function noop () {}`, it will reduce the bundle a bit by not putting in extra module boilerplate.